doc/newsboat.asciidoc

```diff
@@ -834,6 +834,7 @@ Attribute:Context:Meaning
 [[attr-total_count]]<<attr-total_count,+total_count+>>:feed, article:total number of articles in the feed
 [[attr-tags]]<<attr-tags,+tags+>>:feed, article:space-separated list of tags that are associated with the feed. Note that for tags that have spaces in them, it's impossible to tell where they start and end, so +#+ and +!#+ operators don't work for such tags.
 [[attr-feedindex]]<<attr-feedindex,+feedindex+>>:feed, article:Index of a feed in the feed list
+[[attr-latest_article_age]]<<attr-latest_article_age,+latest_article_age+>>:feed, article:Most recent article publish date in feed (in days)
```

Comment:
Somehow my comment was lost during the review:

I reckon the parentheses around "in days" can be removed both here and also the `age` attribute. But maybe even rephrase this to something like that:

Number of days since the most recent article in a feed was published

Reply:
Nice suggestion, thanks!